nipype/interfaces/cat12/preprocess.py

```diff
@@ -239,33 +240,71 @@ class CAT12SegmentInputSpec(SPMCommandInputSpec):
     neuromorphometrics = traits.Bool(
         True,
         field="output.ROImenu.atlases.neuromorphometrics",
-        usedefault=True,
+        # usedefault=True,
```

Comment:
We can't change the interface like this. The reason to have defaults expliciltly added is so that when tools change their defaults, old workflows that depended on those defaults would continue using the old settings.

If the changes are this extensive, new interfaces will be needed.